tests/functional/tlsprofile/test_metrics_exporter_tlsprofile.py

```
@@ -0,0 +1,233 @@
+
+    Skips on FIPS (PQ / ChaCha in our cipher lists). Deletes the CR at the
+    end—only run where that is safe. An autouse fixture also deletes a leftover
+    ``ocs-tls-profile`` if the test aborts before the in-test delete. Exporter
+    pods may roll when TLS settings change.
+    """
+
+    @pytest.fixture(autouse=True)
+    def cleanup_tlsprofile(self, request):
+        tls = TLSProfile()
+
+        def _cleanup():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_cleanup`

Add return type annotation: `None`

(ANN202)